docs/pages/guides/docs.md

```diff
@@ -147,9 +147,10 @@ several good extensions:
 We are including both possible file extensions. We are also avoiding some common
 file patterns, just in case.
 
-For theme, you have several good options. The clean, light-weight Furo theme is
-shown above. Many scientific packages choose the `sphinx-py-data` theme, which
-is also a good choice (no dark mode, though).
+For theme, many scientific packages choose the
+[pydata-sphinx-theme](https://pydata-sphinx-theme.readthedocs.io/). The
+[Furo theme](https://pradyunsg.me/furo/) is another popular choice. The site
+[sphinx-themes.org](https://sphinx-themes.org/) can be used to compare options.
```

Comment:
I also tried to change the `html_theme` to `pydata_sphinx_theme` in the example `conf.py` above, but it appears that pre-commit CI has undone that change. IMO, PST is mature enough now to be shown as the default example. Any guidance on how to get that change to stick?

Reply:
It's from the cookiecutter. The docs are in sync with the template.

I'd rather like a choice if possible, furo is a fantastic, lightweight theme that is very popular for non-scientific libraries. Fine to have the default swap, though.

Reply:
https://github.com/scientific-python/cookie/blob/main/%7B%7Bcookiecutter.project_name%7D%7D/docs/conf.py